torchaudio/functional/functional.py

```diff
@@ -135,6 +136,95 @@ def spectrogram(
     return spec_f
 
 
+def inverse_spectrogram(
+        spectrogram: Tensor,
+        length: Optional[int],
+        pad: int,
+        window: Tensor,
+        n_fft: int,
+        hop_length: int,
+        win_length: int,
+        normalized: bool,
+        center: bool = True,
+        pad_mode: str = "reflect",
+        onesided: bool = True,
+        return_complex: bool = False,
+) -> Tensor:
+    r"""Create an inverse spectrogram or a batch of inverse spectrograms from the provided
+    complex-valued spectrogram.
+
+    Args:
+        spectrogram (Tensor): Complex tensor of audio of dimension (..., freq, time).
+            Alternatively, the tensor can be pseudo-complex, i.e. a real tensor of
+            dimension (..., freq, time, 2), if return_complex is False, although this
+            behavior is deprecated.
+        length (int): The amount to trim the signal by (i.e. the original signal length).
```

Comment:
```suggestion
        length (int, optional): The amount to trim the signal by (i.e. the original signal length).
```

Reply:
I found the expression `The amount to trim the signal by` somewhat confusing. Can we simply say `the output length`?